libopaecxx/src/properties.cpp

```diff
@@ -38,6 +38,7 @@ properties::properties()
     : props_(nullptr),
       type(&props_, fpgaPropertiesGetObjectType, fpgaPropertiesSetObjectType),
       num_errors(&props_, fpgaPropertiesGetNumErrors, fpgaPropertiesSetNumErrors),
+      segment(&props_, fpgaPropertiesGetSegment, fpgaPropertiesSetSegment),
```

Comment:
I don't see where these methods are defined.  Is that a different commit?

Reply:
which methods? fpgaProperties[GS]etSegment?